modin/experimental/engines/pandas_on_ray/sql.py

```diff
@@ -184,6 +184,8 @@ def get_query_info(sql, con, partition_column):
         cols = get_query_columns(engine, query)
     # TODO allow validation that takes into account edge cases of pandas e.g. "[index]"
     # check_partition_column(partition_column, cols)
+    # TODO partition_column isn't used; we need to either use or remove it;
```

Comment:
Do we have an issue open for this? It should be used to determine the partitioning.

Reply:
Created - https://github.com/modin-project/modin/issues/3050